src/main/java/me/eccentric_nz/TARDIS/travel/TARDISPluginRespect.java

```diff
@@ -66,18 +67,49 @@ public boolean getRespect(Location l, Parameters flag) {
                 bool = false;
             }
         }
-        if (flag.permsNether() && !plugin.getConfig().getBoolean("travel.nether") && l.getWorld().getEnvironment().equals(Environment.NETHER)) {
-            if (flag.messagePlayer()) {
+
```

Comment:
Will need to wrapped in:
```
if (flag.permsNether()) {
    // do nether checking
}
```
and
```
if (flag.permsTheEnd()) {
    // do the end checking
}
```
plus all messaging will need to wrapped with:
```
if (flag.messagePlayer()) {
    // message player
}
```
Otherwise it's breaking the contract set by TARDIS API

Reply:
Ah sorry, I didn't look that closely at the code I removed. Thanks for the heads up!